OHHTTPStubs/Sources/Swift/OHHTTPStubsSwift.swift

```diff
@@ -161,8 +191,16 @@ public func isPath(path: String) -> OHHTTPStubsTestBlock {
  * - Note: URL paths are usually absolute and thus starts with a '/' (which you
  *         should include in the `path` parameter unless you're testing relative URLs)
  */
-public func pathStartsWith(path: String) -> OHHTTPStubsTestBlock {
-    return { req in req.URL?.path?.hasPrefix(path) ?? false }
+public func pathStartsWith(_ path: String) -> OHHTTPStubsTestBlock {
+#if swift(>=3.0)
+    return { req in req.url?.path.hasPrefix(path) ?? false }
+#else
+#if os(tvOS)
+    return { req in req.url?.path?.hasPrefix(path) ?? false }
+#else
+    return { req in req.url?.path?.hasPrefix(path) ?? false }
+#endif
```

Comment:
I think 81017a6be3596ad16651bb5a522b7338fe63d984 fixes this but I forgot to remove this conditional 😊